gordo_components/server/server.py

```diff
@@ -123,10 +133,77 @@ def base_healthcheck():
     return app
 
 
-def run_server(host: str = "0.0.0.0", port: int = 5555, debug: bool = False):
-    app = build_app()
-    app.run(host, port, debug=debug, threaded=False)
+def run_server(
+    host: str, port: int, workers: int, worker_connections: int, log_level: str
+):
+    """
+    Run application with Gunicorn server using Gevent Async workers
+
+    Parameters
+    ----------
+    host: str
+        The host to run the server on.
+    port: int
+        The port to run the server on.
+    workers: int
+        The number of worker processes for handling requests.
+    worker_connections: int
+        The maximum number of simultaneous clients per worker process.
+    log_level: str
+        The log level for the server.
```

Comment:
If these are different than the valid ones for `logger` module, then perhaps we should mention what are the valid values, or where to find them?